src/content/community/rollup-grants/index.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Ethereum Rollup Community Grants
+description: Grants to build a rollup ecosystem
+lang: en
+sidebar: true
+---
+
+# Rollup Community Grants {#rollup-community-grants}
+
+_The Ethereum Foundation is sponsoring a wave of rollup community grants. Proposals are due **[date]**. Here are all the details you need._
+
+<Divider />
+
+Calling all those interested in building up the Ethereum rollup community!
+
+The Ethereum ecosystem is likely to be all-in on rollups as a scaling strategy for the near and mid-term future (see Vitalik’s rollup centric ethereum [roadmap](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698/50?u=haoyuathz)). A variety of optimistic and zk-rollups are already in production or will go live in the near future.
+
+As gas prices are soaring the most promising near term solution to getting a scalable EVM like developer experience is optimistic rollups such as [Optimism](https://optimism.io/) and [Arbitrum](https://offchainlabs.com/). Therefore, while we are equally excited about the promise of other rollups, we want to focus this round on EVM optimistic rollup.
+
+The goal is to grow the rollup community and help to kickstart a flourishing ecosystem. We want to make it easy for dapps to live on multiple EVM rollups and have user and developer experience for rollups on par with mainnet or better. Social recovery wallets could be made the default and it should be easy for users to move from one rollup to another even during prolonged periods of very high gas prices.
+
+## Wait, what are optimistic rollups? {#what-are-optimistic-rollups}
+
+Here some links to dive deeper:
+
+- Learn the basics on [Ethhub](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/optimistic_rollups/)
```

Comment:
Please add a link to our page on rollups: `/developers/docs/layer-2-scaling/#rollups`

Reply:
Done :)